src/components/feed/FeedTree/FeedGraph.tsx

```diff
@@ -9,13 +9,16 @@ import {PluginInstance} from '@fnndsc/chrisapi'
 import {ErrorBoundary} from "react-error-boundary";
 import {Text} from '@patternfly/react-core'
 import './FeedTree.scss';
-
+import { Button } from "@patternfly/react-core";
```

Comment:
Just a small nit here.

You can add the `Button` import alongside the `Text` that's being imported above .
```javascript
import {Text, Button} from '@patternfly/react-core';
```
Apart from that, everything lgtm.